cmd/describe/cluster/cmd.go

```diff
@@ -166,7 +167,8 @@ func run(_ *cobra.Command, argv []string) {
 		"State:                      %s %s\n"+
 		"Channel Group:              %s\n"+
 		"Created:                    %s\n",
-		cluster.Name(),
+		cluster.DisplayName(),
```

Comment:
Can you change this so that we check if `DisplayName` is populated, otherwise fallback to `Name`?

Reply:
Thanks!